nixos/modules/services/network-filesystems/kbfs.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.kbfs;
+
+in {
+
+  ###### interface
+
+  options = {
+
+    services.kbfs = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to mount the Keybase filesystem.";
+      };
+
+      mountPoint = mkOption {
+        type = types.str;
+        default = "/keybase";
+        example = "/home/user/keybase";
+        description = "Mountpoint for the Keybase filesystem.";
+      };
+
+      createMountPoint = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Whether to create the mount point directory at startup time.
+        '';
+      };
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.kbfs;
+        defaultText = "pkgs.kbfs";
+        example = literalExample "pkgs.kbfs";
+        description = "Keybase filesystem derivation to use.";
+      };
+
+      extraFlags = mkOption {
+        type = types.listOf types.str;
+        default = [];
+        example = [
+          "-label kbfs"
+          "-mount-type normal"
+        ];
+        description = ''
+          Additional flags to pass to the Keybase filesystem on launch.
+        '';
+      };
+
+    };
+  };
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+
+    systemd.user.services.kbfs = {
+      description = "Keybase File System";
+      path = [ "/run/wrappers" ];
+      preStart = ''
+      ${optionalString cfg.createMountPoint
+      "test -d ${cfg.mountPoint} || mkdir -p ${cfg.mountPoint}"
+      }'';
+      postStop = "fusermount -u ${cfg.mountPoint}";
```

Comment:
Works fine for me? I've been using it the past week with no issues.  
With the `exec` call in the `kbfsfuse` binary to `fusermount`, and this `postStop` operation, that's why I have `path = [ "/run/wrappers" ];` defined.  

Have you actually tested and confirmed this is the case? Because, if so, you'd actually see that any parameters for a systemd service that execute something (`preStart`, `ExecStart`, `postStop`, etc.) are in fact written to a shell script in the nix store, then the resulting systemd unit actually uses a full path to these scripts:
```
[Unit]
Description=Keybase File System

[Service]
Environment="LOCALE_ARCHIVE=/nix/store/mry6hpzxg44idk6pmqhcjvxzr665y43p-glibc-locales-2.25/lib/locale/locale-archive"
Environment="PATH=/run/wrappers/bin:/nix/store/k8dyhz9hrjydl23ny3131am6cws1rc07-coreutils-8.26/bin:/nix/store/dqzqrrampaf665laggc4ng33dfbxv5q7-findutils-4.6.0/bin:<*SNIP*>
Environment="TZDIR=/nix/store/ib356ccx25xixzjmj0fmgdv61kj1f398-tzdata-2016j/share/zoneinfo"



ExecStart=/nix/store/sqv84hq2j879z92pg71h2pmjr0kg58cc-kbfs-20170209.d1db463-bin/bin/kbfsfuse -label kbfs /home/cmacrae/keybase
ExecStartPre=/nix/store/n44y06vk1b18knzng7w9dswkr0dnbm38-unit-script/bin/kbfs-pre-start
ExecStopPost=/nix/store/0ag4fi7jgh1x23y3f7mxpvvihhkbqza5-unit-script/bin/kbfs-post-stop
Restart=always
RestartSec=3
```